src/js/components/stock-movement-wizard/modals/AdjustInventoryModal.jsx

```diff
@@ -178,16 +178,9 @@ class AdjustInventoryModal extends Component {
       };
     });
 
-    return apiClient.post(url, payload).then(() => {
-      apiClient.get(`/openboxes/api/stockMovements/${this.state.attr.stockMovementId}?stepNumber=4`)
-        .then((resp) => {
-          const { pickPageItems } = resp.data.data.pickPage;
-          this.props.onResponse(pickPageItems);
-
-          this.props.hideSpinner();
-        })
-        .catch(() => { this.props.hideSpinner(); });
-    }).catch(() => { this.props.hideSpinner(); });
+    apiClient.post(url, payload)
+      .then(() => { this.state.attr.onResponse(); })
```

Comment:
Is it intentional that you are not passing here item to onResponse? As I see, onResponse from EditPage takes one param (https://github.com/openboxes/openboxes/pull/894/files#diff-f73fcfa54e019f19a48fa8893fbf0642R448).

Reply:
NVM, I looked at wrong page, my mistake